modules/log/config/install/views.view.commerce_activity.yml

```diff
@@ -337,7 +337,7 @@ display:
             type: none
             fail: 'not found'
           validate_options: {  }
-          break_phrase: false
+          break_phrase: true
```

Comment:
This let's us pass in multiple entity id arguments to the source id contextual argument.